graphql/fetchUserProfile.ts

```diff
@@ -30,3 +30,28 @@ export const FETCH_USER_PROFILE = gql`
     }
   }
 `;
+
+export const FETCH_USER_PROFILE_METADATA = gql`
```

Comment:
@jasonrajasegaram you might want to work off this query to fetch user badge data for the user profile.